modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc

```diff
@@ -54,21 +67,19 @@ After you install Anypoint Service Mesh, you can use the command line interface
 image::service-mesh-cli-adapter-help.png[CLI Help for Adapter Commands]
 
 [[provision_the_adapter]]
-== Provisioning the adapter
+== Provisioning the Adapter Using CLI
```

Comment:
We also need to add the replicas parameter to the table params and command.

Reply:
Added a comment. Thanks!